pyaedt/application/Design.py

```diff
@@ -3921,3 +3921,34 @@ def set_temporary_directory(self, temp_dir_path):
         """
         self.odesktop.SetTempDirectory(temp_dir_path)
         return True
+
+    @pyaedt_function_handler()
+    def design_settings(self):
+        """Get design settings for the current AEDT app.
+
+        Returns
+        -------
+        dict
+            Dictionary of valid design settings.
+
+        References
+        ----------
+
+        >>> oDesign.GetChildObject("Design Settings")
+        """
+        try:
+            design_settings = self._odesign.GetChildObject("Design Settings")
+        except Exception:  # pragma: no cover
+            self.logger.error("Could not execute GetChildObject('Design Settings').")
```

Comment:
I would use a more general message. Like: 
`"Failed to retrieve design settings."`

But that's up to you.

Reply:
@lorenzovecchietti @nnetake agree with Loreno here.